torchaudio/transforms/_transforms.py

```diff
@@ -310,6 +310,11 @@ class AmplitudeToDB(torch.nn.Module):
             power being the elementwise square of the magnitude. (Default: ``'power'``)
         top_db (float or None, optional): minimum negative cut-off in decibels.  A reasonable
             number is 80. (Default: ``None``)
+
+    Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
```

Comment:
could you change the single quotes to double quotes, same as the melspectrogram PR?

Reply:
Fix is ready!